devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Install dev packages
+  dnf:
+      name: "{{ item }}"
+      state: present
+  with_items:
+      - gcc
+      - postgresql-devel
+      - python
+      - python-devel
+      - python-alembic
+      - python-psycopg2
+      - redhat-rpm-config
+
+- name: Install python packages
+  pip:
+      name: "{{ item }}"
+      state: present
+  with_items:
+      - kitchen
+      - paver
+      - urllib3
```

Comment:
I personally recommend just putting these packages with the ones above so it becomes one dnf transaction instead of two, only because it'll go a little faster. It's just a preference thing, so if you like this better feel free.